dev-client/src/screens/SoilScreen/components/EditSiteSoilDepthPreset.tsx

```diff
@@ -63,27 +63,27 @@ export const EditSiteSoilDepthPreset = ({selected, updateChoice}: Props) => {
             ),
         }}
       />
-      <ConfirmModal
-        trigger={onOpen => (
-          <Button
-            size="lg"
-            onPress={() => {
-              if (selectedPreset === selected) {
-                modalHandle?.onClose();
-              } else {
-                onOpen();
-              }
-            }}
-            _text={{textTransform: 'uppercase'}}
-            alignSelf="flex-end">
-            {t('general.save')}
-          </Button>
-        )}
-        handleConfirm={onConfirm}
-        title={t('projects.inputs.depths.confirm_preset.title')}
-        body={t('projects.inputs.depths.confirm_preset.body')}
-        actionLabel={t('projects.inputs.depths.confirm_preset.confirm')}
-      />
+      <View alignSelf="flex-end">
```

Comment:
Nit: I might just wrap ContainedButton with this to clarify we're only trying to align the button, but seems fine here too